### PR TITLE
SHOR-47: Markup fixture for Direct Debit forms

### PR DESCRIPTION
**Pull request by swastikpareek**
## Overview
PR fixes HTML markup for Direct Debit Extension FORMS
* Create New Direct Debit Instructions
* Create Direct Debit Payments
* View New Instruction Batches
* Direct Debit Configurations

#### Vanilla CiviCRM
create new direct debit instructions - initial - vanila civicrm
create new direct debit instructions - final - vanila civicrm

#### Shoreditch
create new direct debit instructions - initial - shoreditch  
create new direct debit instructions - final - shoreditch

#### Vanilla CiviCRM
direct debit configurations - initial - vanila civicrm
direct debit configurations - final - vanila civicrm

#### Shoreditch
direct debit configurations - initial - shoreditch
direct debit configurations - final - shoreditch

### Changes in Commit f5d9470 and 453b41d
* Fix Markup for *New Direct Debit Instructions Batches* Page

 #### Before/After
 
direct debit payment batches - previous
new direct debit transaction batch

* Fix Markup for *Create New Direct Debit Instructions* Page
  
  #### Before/After

  new direct debit instructions step2 - previous
  new batch transaction - step 2

## Technical Details
Screens with table format didn't change their styling after fixing the markup (since it is already having bare default markup needed) but I have added other default css classes needed to make it look like default CiviCRM markup.

### CODE REVIEW

File: templates/CRM/ManualDirectDebit/Form/Configurations.tpl [hunk 1]
```diff
@@ -1,53 +1,61 @@
...
+    </tbody>
+  </table>
+  <h3>{ts}Payment config{/ts}</h3>
+  <table class="form-layout-compressed">
+    <tbody>
+      {foreach from=$paymentConfigSection item=elementName}
+        <tr>
+          <td class="label">{$form.$elementName.label}</td>
+          <td>{$form.$elementName.html}
+              {if $fieldsWithHelp.$elementName}{help id=$form.$elementName.name}{/if}
```
Comment (AkA84) [inline templates/CRM/ManualDirectDebit/Form/Configurations.tpl]: indentation is inconsistent

File: templates/CRM/ManualDirectDebit/Form/Configurations.tpl [hunk 2]
```diff
@@ -1,53 +1,61 @@
...
+          </td>
+        </tr>
+      {/foreach}
+    </tbody>
+  </table>
+  <h3 class="title">{ts}Batch and code transition{/ts}</h3>
+  <table style="width:100%" class="row-highlight">
+      <tr class="crm-admin-options crm-entity odd-row">
+          <th></th>
```
Comment (AkA84) [inline templates/CRM/ManualDirectDebit/Form/Configurations.tpl]: indentation is inconsistent

File: templates/CRM/ManualDirectDebit/Form/Configurations.tpl [hunk 3]
```diff
@@ -1,53 +1,61 @@
...
+      </tr>
+      <tr class="crm-admin-options crm-entity even-row">
+          <td>{ts}Codes for inactivity{/ts} {help id="inactivity_code"}</td>
+          <td>0C</td>
+          <td></td>
+          <td></td>
+      </tr>
+  </table>
+  <div class="crm-submit-buttons">
+      {include file="CRM/common/formButtons.tpl" location="bottom"}
```
Comment (AkA84) [inline templates/CRM/ManualDirectDebit/Form/Configurations.tpl]: indentation is inconsistent

File: templates/CRM/ManualDirectDebit/Form/Configurations.tpl (hunk 1 above)
Comment (swastikpareek) [reply templates/CRM/ManualDirectDebit/Form/Configurations.tpl]: Done.

File: templates/CRM/ManualDirectDebit/Form/Configurations.tpl (hunk 2 above)
Comment (swastikpareek) [reply templates/CRM/ManualDirectDebit/Form/Configurations.tpl]: Done.

File: templates/CRM/ManualDirectDebit/Form/Configurations.tpl (hunk 3 above)
Comment (swastikpareek) [reply templates/CRM/ManualDirectDebit/Form/Configurations.tpl]: Done.

**Comment by guanhuan** [comment]
@swastikpareek looks all approved! Please merge.

**Comment by swastikpareek** [comment]
I don't have access. Can you please provide me with the access.
Sorry for the delayed response.

**Pull request merged**